app/controllers/api/v1/lobbies_controller.rb

```diff
@@ -24,6 +24,54 @@ def from_code
         render :show
       end
 
+      def start
+        @lobby = Lobby.find(params[:id])
+
+        if @lobby.update(status: :in_progress)
+          Pusher.trigger(@lobby.code, Lobby::LOBBY_START, {})
+          render :show
+        else
+          render 'api/v1/model_errors', locals: { errors: @lobby.errors }, status: :unprocessable_entity
+        end
+      end
+
+      def answer
+        lobby = Lobby.find(params[:id])
+        player = Player.find_by(id: params[:player_id])
+        return head :not_found if player.nil?
+
+        delete_player_answers(lobby, player)
+        create_player_answer_model(player)
+        send_answer_count_to_pusher(lobby)
+        render :answer
+      end
+
+      def delete_player_answers(lobby, player)
+        current_question = lobby.quiz.questions.find_by(order: lobby.current_question_index)
+        @answers = current_question.answers
+        current_question.answers.each do |answer|
+          PlayerAnswer.where(player: player, answer_id: answer.id).delete_all
+        end
+      end
+
+      def create_player_answer_model(player)
+        answers_ids = params[:answers]
+        answers_ids.each do |answer_id|
+          answer = Answer.find(answer_id)
+          PlayerAnswer.create(player: player, answer: answer)
+        end
+      end
+
+      def send_answer_count_to_pusher(lobby)
```

Comment:
This method is chronologically conditioned by `delete_player_answers`. This usually leads to bad code practices, especially since neither `delete_player_answers` nor `send_answer_count_to_pusher` seem to have any tie with each other. 

A better way to do it would be to either have `@answers` set in a `before_action`, or simply assign it here since it's just a matter of doing `answers = lobby.quiz.questions.find_by(order: lobby.current_question_index).answers`